tools/db_bench_tool.cc

```diff
@@ -4667,23 +4667,26 @@ void VerifyDBFromDB(std::string& truth_db_name) {
     int64_t seek = 0;
     int64_t seek_found = 0;
     int64_t bytes = 0;
-    int64_t value_max = FLAGS_mix_max_value_size;
+    int64_t value_max = 64*1024*1024;
     int64_t scan_len_max = FLAGS_mix_max_scan_len;
     double write_rate = 1000000.0;
     double read_rate = 1000000.0;
-    std::vector<double> ratio;
-    char value_buffer[2 * value_max];
+    std::vector<double> ratio(3);
+    char value_buffer[65*1024*1024];
```

Comment:
It would be nice if we pulled in those numbers into some const. may be like:
```
const int64_t default_value_max = 64*1024*1024;
int64_t value_max = default_value_max;
char value_buffer[default_value_max]; 
```
I believe `value_buffer[default_value_max]` will not be considered as a variable length array as `default_value_max` is const and initialized.

Reply:
Sure